gson/src/main/java/com/google/gson/internal/$Gson$Preconditions.java

```diff
@@ -34,13 +34,6 @@
     throw new UnsupportedOperationException();
   }
 
-  public static <T> T checkNotNull(T obj) {

```

Comment:
To my distress I found 3 places in Google's code base that are calling this method, even though it's hard to imagine how it could be more obviously internal. I can fix these, but on the assumption that code elsewhere is doing the same thing, maybe leave the method intact.

Reply:
I'm guessing BTW that these happened because an IDE auto-imported the wrong `checkNotNull`. Which can easily happen to other projects.